arrayfire/array.py

```diff
@@ -1194,6 +1252,9 @@ def to_string(self):
         You can also use af.display(a, pres) to display the contents of the array with better precision.
         """
 
+        if not _in_display_dims_limit(self.dims()):
+            return self.__repr__();
```

Comment:
Can we have a common function that is called from both `__repr__` and `__str__` instead of calling `self.__repr__` here?